jha/src/components/enrolment/Child.vue

```diff
@@ -142,7 +142,7 @@
 
       <div v-if="requestImmigrationStatus">
         <h2 class="mt-3">Child's status in Canada</h2>
-        <p>Please provide your child's immigration status. You will be required to upload documents to support your child's status in Canada. For arrivals through the Canada-Ukraine Authorization for Emergency Travel (CUAET) program, please select "Temporary Permit Holder or Diplomat" from the menu below.</p>
+        <p>Please provide your child's immigration status. You will be required to upload documents to support your child's status in Canada. For arrivals through the Canada-Ukraine authorization for emergency travel (CUAET) program, please select "Temporary Permit Holder or Diplomat" from the menu below.</p>
```

Comment:
Usually words that are followed by an acronym like this stay capitalized to make it clear which words make up the acronym. Are we able to do that here?

Reply:
They specifically requested it be lowercase. I have no idea why but I'm not going to argue